src/rtk/features/onBoarding/onBoardingHooks.ts

```diff
@@ -106,9 +105,9 @@ const getSavedProfileData = (profile: SpaceContent | undefined) => {
   return { about, email, image, links, name, tags }
 }
 const upsertProfile = (api: ApiPromise, cid: IpfsCid, currentProfileId?: string) => {
-  const permissions = newWritePermission()
   if (!currentProfileId) {
-    return api.tx.spaces.createSpace(IpfsContent(cid?.toString()), permissions)
+    // TODO: remove any cast when sdk is updated
```

Comment:
Create task and link with it
https://app.clickup.com/t/8677d8xh5

Reply:
Created the PR in https://github.com/dappforce/subsocial-js/pull/127